utils/ActionListeners.js

```diff
@@ -1,62 +0,0 @@
-'use strict'
```

Comment:
I was hoping we'd keep these here so we don't break linking:

`require('alt/utils/Util')`

vs

`require('alt/lib/utils/Util')`